src/microbe_stage/PlayerHoverInfo.cs

```diff
@@ -29,7 +32,42 @@ public void Init(MicrobeCamera camera, CompoundCloudSystem cloudSystem)
 
     public override void _Process(float delta)
     {
-        HoveredCompounds = cloudSystem.GetAllAvailableAt(camera.CursorWorldPos);
```

Comment:
Might be good to check against negative delta here, but I don't see anything in here that would really break if the delta is occasionally negative in here, so up to you if you want to add that or not.

```
        // https://github.com/Revolutionary-Games/Thrive/issues/1976
        if (delta <= 0)
            return;
```